server/controller/rides.js

```diff
@@ -43,26 +38,23 @@ class RidesController{
     }
 
     static postRide (req, res, next) {
-
-        const text = "INSERT INTO rides(destinationstartpoint, destinationstoppoint, departuretime, userId, created_at) VALUES ($1, $2, $3, $4, Now()) RETURNING *";
+        const text = "INSERT INTO rides(startPoint, stopPoint, departureTime, userId, created_at) VALUES ($1, $2, $3, $4, Now()) RETURNING *";
         const values = [
-            req.body.destinationstartpoint,
-            req.body.destinationstoppoint,
-            req.body.departuretime,
+            req.body.startPoint,
+            req.body.stopPoint,
+            req.body.departureTime,
             req.decoded.userId
         ]
-        //console.log(req)
         client.query(text, values, (err, data) => {
            if(err) return res.status(500).json({
                success: false,
                message: 'Internal Server error',
                error: next(err)
```

Comment:
why did you add next(err)? I think it's suppose to be err.message

Reply:
fixed.